plugins/AudioFileProcessor/AudioFileProcessorWaveView.h

```diff
@@ -158,8 +162,9 @@ public slots:
 	void zoom(const bool out = false);
 	void slide(int px);
 	void slideSamplePointByPx(Point point, int px);
-	void slideSamplePointByFrames(Point point, f_cnt_t frames, bool slide_to = false);
-	void slideSampleByFrames(f_cnt_t frames);
+	// point: wihch knob to slide, frames: how mutch, can be negative, slide_to: should set the value instead of adding it
```

Comment:
```suggestion
```

Reply:
Why do you suggest removing this comment?

Reply:
The comment feels like added noise. If you insist, I would try to clean up the spelling/punctuation/grammar and use `//!` for documentation comments.

Reply:
All right, I wanted to make it clear that `frames` can be negative, this comment will be removed.

Reply:
> All right, I wanted to make it clear that `frames` can be negative, this comment will be removed.

I would suggest using a better name to make it clear that `frames` is not actually for a count of frames, but rather used to determine how many frames should we slide/zoom.

I can't think of one off the top of my head, so don't bother with it if it's too much trouble. You can still use the comment here, just make it a documentation comment and clean it up a bit.